spec/spec_helper.rb

```diff
@@ -51,11 +51,11 @@
     c.syntax = [:expect]
   end
 
-  # This prevents affiliate-related tests from failing with an empty test db
+  # This prevents fixture related fails with an empty test db
```

Comment:
@HalahAb , let's simplify this entire comment to be:
`# Create all fixtures in spec/fixtures/*.yml for all tests in alphabetical order.`

Reply:
should I get rid of the comment about the language fixtures as well? or should we keep it for future reference?